### PR TITLE
Better UI & assets management

### PR DESCRIPTION
**Pull request by florianbelhomme**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,8 +30,9 @@ Website to generate a environment with Docker and Symfony.
 ## Install the developement environment
 
 - docker-compose up -d
-- docker-compose run --rm tool /bin/bash
+- docker-compose run --rm tool bash
 - composer install
```
Comment (romqin) [inline README.md]: indent:

- a
  - b
  - c

File: src/AppBundle/Resources/views/default/index.html.twig [hunk 2]
```diff
@@ -7,15 +7,19 @@
         <div class="tab-content">
             {% for name, element in tabs %}
                 <div class="tab-pane{% if name == activeTab %} active{% endif %}" id="{{ name }}" role="tabpanel">
+                    <h2>{{ 'title.' ~ name|trans }}</h2>
                     {% include '@App/default/partial/_' ~ name ~ '.html.twig' with {
-                        form: form[name] is defined ? form[name] : null
+                        name: name,
+                        form: form[name] is defined ? form[name] : form
```
Comment (romqin) [inline src/AppBundle/Resources/views/default/index.html.twig]: form: form[name] ?? form

File: README.md (hunk 1 above)
Comment (romqin) [inline README.md]: better: 
```
docker-compose run --rm tool composer install
docker-compose run --rm tool npm install
```

File: README.md [hunk 3]
```diff
@@ -30,8 +30,8 @@ Website to generate a environment with Docker and Symfony.
 ## Install the developement environment
 
 - docker-compose up -d
-- docker-compose run --rm tool /bin/bash
-- composer install
+- docker-compose run --rm tool composer install
+- docker-compose run --rm tool npm install
```
Comment (romqin) [inline README.md]: What about gulp ?

File: package.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    "awesome-bootstrap-checkbox": "^0.3.7",
+    "bootstrap": "^3.3.6",
+    "bootswatch": "^3.3.6",
+    "font-awesome": "^4.6.1",
+    "jquery": "^1.12.3",
+    "normalize-css": "^2.3.1",
+    "selectize": "^0.12.1"
+  },
+  "scripts": {
+    "postinstall": "./node_modules/gulp/bin/gulp.js default"
```
Comment (romqin) [inline package.json]: `"postinstall": "./node_modules/.bin/gulp"`

File: src/AppBundle/Resources/views/_form_theme.html.twig [hunk 5]
```diff
@@ -34,3 +34,75 @@
...
+        {% if align_with_widget is defined or attr.align_with_widget is defined %}
+            </div>
+            </div>
+        {% elseif no_form_group is not defined or no_form_group == false %}
+            </div>
+        {% endif %}
+
+        {% if style == 'inline' %}&nbsp;{% endif %}
+    {% endspaceless %}
+{% endblock checkbox_row %}
```
Comment (romqin) [inline src/AppBundle/Resources/views/_form_theme.html.twig]: new line after

File: src/AppBundle/Resources/views/layout.html.twig [hunk 6]
```diff
@@ -4,30 +4,33 @@
...
-{% block title %}ContainSymfony{% endblock %}
+{% block title %}{{ 'page_title.main'|trans }}{% endblock %}
 
 {% block stylesheets %}
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/{{ normalize }}/normalize.min.css">
-    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/{{ boostrap }}/css/bootstrap.min.css">
-    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/font-awesome/{{ fontawesome }}/css/font-awesome.min.css">
-    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/selectize.js/{{ selectize }}/css/selectize.bootstrap3.min.css">
+    {% for vendor in [
+        '/normalize-css/normalize.css',
```
Comment (romqin) [inline src/AppBundle/Resources/views/layout.html.twig]: remove the first / on all

File: src/AppBundle/Resources/views/layout.html.twig [hunk 7]
```diff
@@ -4,30 +4,33 @@
...
+        '/bootswatch/bootstrap.min.css',
+        '/font-awesome/css/font-awesome.min.css',
+        '/selectize/css/selectize.bootstrap3.css',
+        '/awesome-bootstrap-checkbox/awesome-bootstrap-checkbox.css'
+    ] %}
+        <link rel="stylesheet" href="{{ asset('/vendor' ~ vendor) }}">
```
Comment (romqin) [inline src/AppBundle/Resources/views/layout.html.twig]: /vendor/

File: src/AppBundle/Resources/views/layout.html.twig [hunk 8]
```diff
@@ -4,30 +4,33 @@
...
-    <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/{{ jquery }}/jquery.min.js"></script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/{{ boostrap }}/js/bootstrap.min.js"></script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/selectize.js/{{ selectize }}/js/standalone/selectize.min.js"></script>
-    <script src="{{ asset('bundles/app/js/main.js') }}"></script>
+    {% for vendor in [
+        '/jquery/jquery.min.js',
+        '/bootstrap/js/bootstrap.min.js',
+        '/selectize/js/standalone/selectize.min.js'
+    ] %}
+        <script type="text/javascript" src="{{ asset('/vendor' ~ vendor) }}"></script>
```
Comment (romqin) [inline src/AppBundle/Resources/views/layout.html.twig]: same here

File: src/AppBundle/Resources/views/layout.html.twig [hunk 9]
```diff
@@ -4,30 +4,33 @@
...
-
-{% block title %}ContainSymfony{% endblock %}
+{% block title %}{{ 'page_title.main'|trans }}{% endblock %}
 
 {% block stylesheets %}
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/{{ normalize }}/normalize.min.css">
-    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/{{ boostrap }}/css/bootstrap.min.css">
-    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/font-awesome/{{ fontawesome }}/css/font-awesome.min.css">
-    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/selectize.js/{{ selectize }}/css/selectize.bootstrap3.min.css">
+    {% for vendor in [
```
Comment (romqin) [inline src/AppBundle/Resources/views/layout.html.twig]: what about renaming vendor into css ?

File: src/AppBundle/Resources/views/layout.html.twig [hunk 10]
```diff
@@ -4,30 +4,33 @@
...
     <link rel="stylesheet" href="{{ asset('bundles/app/css/main.css') }}">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 {% endblock %}
 
 {% block javascripts %}
-    <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/{{ jquery }}/jquery.min.js"></script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/{{ boostrap }}/js/bootstrap.min.js"></script>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/selectize.js/{{ selectize }}/js/standalone/selectize.min.js"></script>
-    <script src="{{ asset('bundles/app/js/main.js') }}"></script>
+    {% for vendor in [
```
Comment (romqin) [inline src/AppBundle/Resources/views/layout.html.twig]: renaming vendor into js ?

File: src/AppBundle/Resources/views/layout.html.twig (hunk 6 above)
Comment (florianbelhomme) [inline src/AppBundle/Resources/views/layout.html.twig]: path must start by a /

**Comment by florianbelhomme** [comment]
The screenshot is what the site should look.

**Pull request merged**